src/doc/doc-builders.js

```diff
@@ -41,6 +65,11 @@ function align(n, contents) {
   return { type: "align", contents, n };
 }
 
+/**
+ * @param {Doc} contents
+ * @param {Object} [opts] - TBD ???
```

Comment:
I think we should define a new doc options type, ideally in a separate PR. Any guidance would be much appreciated.